### PR TITLE
Updating packages

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Updates DS, Jest, React, Next.

### CODE REVIEW

**Comment by hokei** [comment]
It doesn't seem to work on node 19.  Can you add a thing to the README updating node versions (16 works) and/or a `.nvmrc`?   (I don't think we need to be exhaustive on what is allowed, we just want to give people something that will work)

File: README.md [hunk 1]
```diff
@@ -29,7 +29,9 @@ The main entry file is `pages/index.tsx` but components can be organized in `src
 Prerequisites: `node` (v10 minimum) and `npm` (v6 minimum) or `yarn` (v1.20 minimum)
```
Comment (hokei) [inline README.md]: It doesn't seem to work on node 19. Can we give them a version here that we know will work (or use an .nvmrc) or both?

**Review by hokei — CHANGES_REQUESTED** [review]
The storybook link (https://nypl.github.io/nypl-design-system/storybook-static/?path=/story/introduction--page)  in the readme needs updating -- and I would also check the links in the instructions to make sure they still work

Everything else looks good, thanks for this!

File: README.md (hunk 1 above)
Comment (EdwinGuzman) [reply README.md]: node 19 is so new!!!

Comment (hokei) [reply README.md]: my computer died so I was actually installing on a completely new setup haha

**Review by hokei — APPROVED** [review]

**Pull request merged**